e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -83,11 +83,11 @@ Scenario('Gatekeeper enters hearing details and submits', async (I, caseViewPage
 });
 
 // Disabled until SDO goes live
-xScenario('Gatekeeper drafts standard directions', async (I, caseViewPage, draftStandardDirectionsEventPage) => {
+Scenario('Gatekeeper drafts standard directions', async (I, caseViewPage, draftStandardDirectionsEventPage) => {
```

Comment:
remember to disable again

Reply:
Not needed anymore if we use non prod :D

Reply:
It likely be done in separate PR @akomisarek as this test touches SDO that is not subject of that story.